### PR TITLE
Feature#24 add validation to employee model

### LINKED ISSUE #24: Add validation to employee model
# REQUIREMENTS
Add validation for the following attributes:
- Name (at least 5 chars long)
- Email (must be unique)
- Position (at least 3 chars long)
- Private number (all digits and must be unique)

### PR DESCRIPTION
**Pull request by HikingCarrot7**
# Description
- Added validations for employee model.
- Show validation errors for create and edit forms.

### CODE REVIEW

**Review by lucasvidela94 — APPROVED** [review]
LGTM!

**Review by VictorElizalde — APPROVED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by acorzo — APPROVED** [review]
LGTM

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -11,17 +11,23 @@ def new
 
   def create
     @employee = Employee.create employee_params
-    @employee.save
-    redirect_to employees_path
+    if @employee.save
```
Comment (MatiasDevJam) [inline app/controllers/employees_controller.rb:14]: You can use AR bang method [Link doc](https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html)

Comment (lucasvidela94) [reply app/controllers/employees_controller.rb:14]: Thanks for the advice!We will check the bang methods.

**Review by EdwardMD — APPROVED** [review]
nice commit

**Comment by JuanequeX** [comment]
Remember delete your branches after to merge it. @lucasvidela94

**Pull request merged**